segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/ServiceStarter.java

```diff
@@ -198,6 +213,20 @@ private CuratorFramework createZKClient() {
         return zkClient;
     }
 
+    private String loadPasswordFrom(String zkTrustStorePasswordPath) {
+        byte[] pwd;
+        File passwdFile = new File(zkTrustStorePasswordPath);
+        if (passwdFile.length() == 0) {
+            return "";
+        }
+        try {
+            pwd = FileUtils.readFileToByteArray(passwdFile);
+        } catch (IOException e) {
+            return "";
```

Comment:
you may wan to add a log here. And should we return empty password for all IO exceptions? Or only FileNotFound variety?

Reply:
Please see my comment from the same method you have in the Controller. Since this method is identical to that one, please either create a shared method somewhere or completely replace this with `FileUtils.readFileToString`